src/app/lib/styledGrid/index.jsx

```diff
@@ -27,6 +29,155 @@ import {
   gridContainerLargeCss,
 } from '../layoutGrid';
 
+export const MediumContentContainer = styled.div`
+  @media (min-width: ${GEL_GROUP_4_SCREEN_WIDTH_MIN}) {
+    margin: 0 auto;
+    max-width: 46.4rem;
```

Comment:
What is the reason for needing this very specific max-width value that maps to `742.4px`?

Reply:
Created `MediumContentContainer` to be used as a content wrapper that centres the contents of `ArticlePage`, `PhotoGalleryPage`, `MediaAssetPage` . Currently `46.4` is the max width for the contents of those pages.

Reply:
@sadickisaac I think you can use this existing `GelPageGrid` instead: https://github.com/bbc/simorgh/blob/latest/src/app/components/Grid/index.jsx#L20

Reply:
I am not sure `GelPageGrid` fits the use intended for `ArticlePage`, `PhotoGalleryPage` and `MediaAssetPage`